packages/node_modules/@webex/webex-core/src/lib/services/services.js

```diff
@@ -410,6 +410,15 @@ const Services = WebexPlugin.extend({
     const {idbroker, identity} = this.list(true);
 
     if (idbroker && identity) {
+      const {authorizationString, authorizeUrl} = this.webex.config.credentials;
+
+      // This must be set outside of the setConfig method used to assign the
+      // idbroker and identity url values.
+      this.webex.config.credentials.authorizeUrl = authorizationString ?
```

Comment:
Why do we do this explicitly here and use `setConfig` below?

Reply:
Set config allows the safe mutation of nested object values while maintaining the AmperstandState events:

```js
idbroker: {
  url: `nested object value A` // Only the value changed, object still mutable, event persists
},
identity: {
  url: `nested object value B` // Only the value changed, object still mutable. Event persists
}
```

When attempting to set these values directly, some unwated artifacts occur with eventing. When attempting to set `authorizeUrl` using these values, errors are thrown at runtime / testing.